webrtc.html

```diff
@@ -2184,14 +2184,12 @@
                     <code><var>receiver.rtcpTransport.transport.state</var></code>
                     to "closed".</p>
                   </li>
-                  <li>
-                    <p>All <code><a>RTCRtpSender</a></code>s in
-                    <var>senders</var> are now considered <a>stopped</a>.</p>
-                  </li>
-                  <li>
-                    <p>All <code><a>RTCRtpReceiver</a></code>s in
-                    <var>receivers</var> are now considered <a>stopped</a>.</p>
-                  </li>
+                    <p>Let <var>transceivers</var> be the result of executing the
+                    <code><a>CollectTransceivers</a></code> algorithm. For every
+                    <code><a>RTCRtpTransceiver</a></code> <var>transceiver</var> in
+                    <var>transceivers</var>, set
+                    <code><var>transceiver</var>.stopped</code> to
```

Comment:
This should probably point to the `RTCRtpTransceiver.stop` method, and not just set the attribute directly.